File: examples/automations/endpoints/get.py
from dotenv import load_dotenv
from retab import Retab

assert load_dotenv("../../../.env.production")

reclient = Retab()

automation = reclient.processors.automations.endpoints.get(
    endpoint_id="endp_fJVB60EdGO7XXT5k02cvn",
)

print(automation.model_dump_json(indent=2))

File: examples/typescript/automations/endpoints/get.ts
import { config } from 'dotenv';
import { Retab } from '@retab/node';



async function main() {
  config({ path: '../../../.env.local' });
  
  const reclient = new Retab();
  
  const automation = await reclient.processors.automations.endpoints.get({
    endpoint_id: "endp_jKH0WW6X1dD3wtWbnuuXQ",
  });
  
  console.log(JSON.stringify(automation, null, 2));
  
}

main().catch(console.error);